src/main/java/com/cisco/trex/util/TrexClientUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.cisco.trex.util;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import com.cisco.trex.stateless.TRexCommand;
+import com.cisco.trex.stateless.TRexTransport;
+import com.cisco.trex.stateless.model.RPCResponse;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Util class for Trex Client
+ */
+public final class TrexClientUtil {
```

Comment:
first two letters should be capital
"TRexServerMode"

Reply:
done